internal/kafka/internal/config/dataplane_cluster_config.go

```diff
@@ -85,13 +95,47 @@ func NewDataplaneClusterConfig() *DataplaneClusterConfig {
 			Namespace:              constants.StrimziOperatorNamespace,
 			SubscriptionChannel:    "alpha",
 			Package:                "managed-kafka",
+			SubscriptionConfig: operatorsv1alpha1.SubscriptionConfig{
+				Resources: v1.ResourceRequirements{
+					Requests: v1.ResourceList{
+						v1.ResourceMemory: resource.MustParse("1Gi"),
+						v1.ResourceCPU:    resource.MustParse("500m"),
+					},
+					Limits: v1.ResourceList{
+						v1.ResourceMemory: resource.MustParse("1Gi"),
+						v1.ResourceCPU:    resource.MustParse("500m"),
+					},
+				},
+			},
+			SubscriptionConfigFile: "config/strimzi-operator-subscription-spec-config.yaml",
```

Comment:
I'm not so sure about specifying default values (in code or in YAML). These resources requests/limits and the env variables from the YAML would be applied if nothing is specified by the user's configuration, correct?

Reply:
Yes. If nothing is specified by the user, the default resources limits e.g in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/971/files#diff-fef0e0515f4cddc045b1e59b5302c05fb909a1c85e5aed861b07b2782928a206R98 will be used.

Reply:
> These resources requests/limits and the env variables from the YAML would be applied if nothing is specified by the user's configuration

The user is to supply/configure this via the YAML file. For local development, our config files lives in `config/` folder. 
For when deploying in OpenShift (the case for kas-installer), the value from the ConfigMap, created via the exposed parameters will be used.

When the YAML is empty or does not exist, the default values defined in: 
- https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/195ed09316951c89e6253cb9e3fb80a0c5e489d3/internal/kafka/internal/config/dataplane_cluster_config.go#L98-L109 and
- https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/195ed09316951c89e6253cb9e3fb80a0c5e489d3/internal/kafka/internal/config/dataplane_cluster_config.go#L119-L139

will be used.

Reply:
I'll go ahead and remove the defaults from the code and let everything be configured via the yaml.

However, I'll leave the value defined in `config/*.yaml` file for local development.